bluebrain/repo-bluebrain/packages/py-atlas-direction-vectors/package.py

```diff
@@ -6,31 +6,24 @@
 class PyAtlasDirectionVectors(PythonPackage):
     """Commands to compute direction vectors in volumetric brain regions."""
     homepage = "https://github.com/BlueBrain/atlas-direction-vectors"
-    git      = "https://github.com/BlueBrain/atlas-direction-vectors.git"
-    pypi     = "atlas-direction-vectors/atlas-direction-vectors-0.1.2.tar.gz"
+    git = "https://github.com/BlueBrain/atlas-direction-vectors.git"
+    pypi = "atlas-direction-vectors/atlas-direction-vectors-0.1.2.tar.gz"
 
     version('develop', branch='main')
-    version('0.1.2', tag='d87ff0caa7e03f62fa03fcd3230c398c3f74f5b7f550ac204098befbba6b8f45')
+    version('0.1.4', tag='5c856319e6b24a31f796e94ff62433c7c605d3af2d20de3361ecb8ae12f4033f')
```

Comment:
That's a weird tag...
```suggestion
    version('0.1.4', sha256='5c856319e6b24a31f796e94ff62433c7c605d3af2d20de3361ecb8ae12f4033f')
```
?